.vscode/settings.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I think this is great stuff you added here. But  what about someone who decides to use another editor, our app shouldn't depend only on vscode, what do you think?

Reply:
So, I think this shouldn't fly in.

Reply:
It can be in the docs, what do you think?